test/test-panel.js

```diff
@@ -893,34 +893,34 @@ exports['test passing DOM node as first argument'] = function (assert, done) {
 
   let { loader } = LoaderWithHookedConsole(module, onMessage);
   let { Panel } = loader.require('sdk/panel');
-  let { Widget } = loader.require('sdk/widget');
+  let { ActionButton } = loader.require('sdk/ui/button/action');
+  let { getNodeView } = loader.require('sdk/view/core');
   let { document } = getMostRecentBrowserWindow();
-  let widgetId = 'widget:' + self.id + '-panel-widget';
 
   let panel = Panel({
     onShow: function() {
       let panelNode = document.getElementById('mainPopupSet').lastChild;
 
-      assert.equal(panelNode.anchorNode, widgetNode,
-        'the panel is properly anchored to the widget');
+      assert.equal(panelNode.anchorNode, buttonNode,
+        'the panel is properly anchored to the button');
 
       shown.resolve();
     }
   });
 
-  let widget = Widget({
+  let button = ActionButton({
     id: 'panel-widget',
     label: 'panel widget',
```

Comment:
"widget" is a generic term, is used also in firefox to identify buttons; that's why I didn't update it; but I do not have issue to change label and id, it's fine to me.